### PR TITLE
l10n: uk: update translation

### PR DESCRIPTION
**Pull request by kasya**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -16571,23 +16572,24 @@ msgstr "опції \"%s\", \"%s\" та \"%s\" не можна використо
 
 #: builtin/worktree.c:652
 msgid "added with --lock"
-msgstr ""
+msgstr "додано з --lock"
 
 #: builtin/worktree.c:714
 msgid "--[no-]track can only be used if a new branch is created"
 msgstr ""
+"--[no-]track можна використовувати лише у випадку створення нової гілки"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"--[no-]track можна використовувати лише при створенні нової гілки"
```

File: po/uk.po [hunk 2]
```diff
@@ -16571,23 +16572,24 @@ msgstr "опції \"%s\", \"%s\" та \"%s\" не можна використо
...
 
 #: builtin/worktree.c:836
 msgid "add 'prunable' annotation to worktrees older than <time>"
-msgstr ""
+msgstr "додати анотацію \"prunable\" до робочих дерев, старших за <час>"
 
 #: builtin/worktree.c:838
 msgid "terminate records with a NUL character"
-msgstr ""
+msgstr "завершити записи з символом NUL"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "завершувати записи символом NUL"
```

File: po/uk.po [hunk 3]
```diff
@@ -16617,10 +16619,12 @@ msgstr "\"%s\" не зафіксовано"
 #: builtin/worktree.c:967
 msgid "working trees containing submodules cannot be moved or removed"
 msgstr ""
+"робочі дерева, що містять підмодулі, не можуть бути переміщені або видалені"
 
 #: builtin/worktree.c:975
 msgid "force move even if worktree is dirty or locked"
 msgstr ""
+"примусове переміщення, навіть якщо робоче дерево забруднене або заблоковане"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"перемістити примусово, навіть якщо робоче дерево забруднене або заблоковане"
```

File: po/uk.po [hunk 4]
```diff
@@ -16630,25 +16634,31 @@ msgstr "\"%s\" є головним робочим деревом"
...
 msgstr ""
+"неможливо перемістити заблоковане робоче дерево;\n"
+"скористайтесь командою \"move -f -f\", щоб спочатку перевизначити або "
+"розблокувати його"
 
 #: builtin/worktree.c:1022
 #, c-format
 msgid "validation failed, cannot move working tree: %s"
-msgstr ""
+msgstr "валідація не пройшла, не вдається перемістити робоче дерево: %s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "валідація завершилася невдало, неможливо перемістити робоче дерево: %s"
```

File: po/uk.po [hunk 5]
```diff
@@ -16673,54 +16685,59 @@ msgstr "не вдалося виконати \"git status\" на \"%s\", код
 #: builtin/worktree.c:1105
 msgid "force removal even if worktree is dirty or locked"
 msgstr ""
+"примусове видалення, навіть якщо робоче дерево забруднене або заблоковане"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"видалити примусово, навіть якщо робоче дерево забруднене або заблоковане"
```

File: po/uk.po [hunk 6]
```diff
@@ -16673,54 +16685,59 @@ msgstr "не вдалося виконати \"git status\" на \"%s\", код
...
 #, c-format
 msgid "validation failed, cannot remove working tree: %s"
-msgstr ""
+msgstr "валідація завершилася невдало, неможливо видалити робоче дерево: %s"
 
 #: builtin/worktree.c:1157
 #, c-format
 msgid "repair: %s: %s"
-msgstr ""
+msgstr "ремонт: %s: %s"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "відремонтувати: %s: %s"
```

File: po/uk.po [hunk 7]
```diff
@@ -16673,54 +16685,59 @@ msgstr "не вдалося виконати \"git status\" на \"%s\", код
...
 
 #: builtin/write-tree.c:33
 msgid "write tree object for a subdirectory <prefix>"
-msgstr ""
+msgstr "записати обʼєкт дерева для піддиректорії <префікс>"
 
 #: builtin/write-tree.c:35
 msgid "only useful for debugging"
-msgstr ""
+msgstr "корисно лише для відлагоджування"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "корисно лише для відлагодження"
```

File: po/uk.po [hunk 8]
```diff
@@ -16767,28 +16784,28 @@ msgstr "не вдалося завантажити пакунок з URI \"%s\""
 #: bundle-uri.c:725
 #, c-format
 msgid "file at URI '%s' is not a bundle or bundle list"
-msgstr ""
+msgstr "файл за URI \"%s\" не є пакунком або списком пакунків"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "файл з URI \"%s\" не є пакунком або списком пакунків"
```

File: po/uk.po [hunk 9]
```diff
@@ -16803,46 +16820,48 @@ msgstr "невідома властивість \"%s\""
...
 #, c-format
 msgid "'%s' does not look like a v2 or v3 bundle file"
-msgstr ""
+msgstr "\"%s\" не схожий на файл пакунка v2 або v3"
 
 #: bundle.c:123
 #, c-format
 msgid "unrecognized header: %s%s (%d)"
-msgstr ""
+msgstr "unrecognized header: %s%s (%d)"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "нерозпізнаний заголовок: %s%s (%d)"
```

File: po/uk.po [hunk 10]
```diff
@@ -16803,46 +16820,48 @@ msgstr "невідома властивість \"%s\""
...
 
 #: bundle.c:220
 msgid "Repository lacks these prerequisite commits:"
-msgstr ""
+msgstr "У сховищі не вистачає цих необхідних комітів:"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "У сховищі не вистачає обовʼязкових комітів:"
```

File: po/uk.po [hunk 11]
```diff
@@ -16854,26 +16873,26 @@ msgstr "Не вдалося розмножити об’єкти пакункі
 
 #: bundle.c:362
 msgid "pack-objects died"
-msgstr ""
+msgstr "отримання обʼєктів пакунка завершилось невдало"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "pack-objects завершився невдало"
```

File: po/uk.po [hunk 12]
```diff
@@ -16882,26 +16901,26 @@ msgstr "неможливо створити \"%s\""
 
 #: bundle.c:637
 msgid "index-pack died"
-msgstr ""
+msgstr "отримання індексного пакунка завершилось невдало"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "index-pack завершився невдало"
```

File: po/uk.po [hunk 13]
```diff
@@ -16882,26 +16901,26 @@ msgstr "неможливо створити \"%s\""
...
 
 #: chunk-format.c:121
 msgid "terminating chunk id appears earlier than expected"
-msgstr ""
+msgstr "ідентифікатор завершення фрагмента зʼявляється раніше, ніж очікувалося"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "ідентифікатор завершення фрагмента зʼявився раніше, ніж очікувалося"
```

File: po/uk.po [hunk 14]
```diff
@@ -16882,26 +16901,26 @@ msgstr "неможливо створити \"%s\""
...
 #, c-format
 msgid "duplicate chunk ID %<PRIx32> found"
-msgstr ""
+msgstr "знайдено дубльований шматок з ідентифікатором %<PRIx32>"
 
 #: chunk-format.c:151
 #, c-format
 msgid "final chunk has non-zero id %<PRIx32>"
-msgstr ""
+msgstr "кінцевий шматок має ненульовий ідентифікатор %<PRIx32>"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "заключний шматок має ненульовий ідентифікатор %<PRIx32>"
```

File: po/uk.po [hunk 15]
```diff
@@ -16914,163 +16933,165 @@ msgstr "неприпустиме значення кольору: %.*s"
...
 
 #: command-list.h:59
 msgid "Create an archive of files from a named tree"
-msgstr ""
+msgstr "Створити архів файлів з названого дерева"
 
 #: command-list.h:60
 msgid "Use binary search to find the commit that introduced a bug"
-msgstr ""
+msgstr "Використовуйте бінарний пошук, щоб знайти коміт, який вніс помилку"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Використати бінарний пошук, щоб знайти коміт, який вніс помилку"
```

File: po/uk.po [hunk 16]
```diff
@@ -16914,163 +16933,165 @@ msgstr "неприпустиме значення кольору: %.*s"
...
+msgstr "Показати, яка ревізія та автор востаннє змінювали кожен рядок файлу"
 
 #: command-list.h:62
 msgid "List, create, or delete branches"
 msgstr "Показати, створити або видалити гілки"
 
 #: command-list.h:63
 msgid "Collect information for user to file a bug report"
-msgstr ""
+msgstr "Збирати інформацію, щоб користувач міг подати звіт про помилку"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Зібрати інформацію, щоб користувач міг подати звіт про помилку"
```

File: po/uk.po [hunk 17]
```diff
@@ -16914,163 +16933,165 @@ msgstr "неприпустиме значення кольору: %.*s"
...
 
 #: command-list.h:64
 msgid "Move objects and refs by archive"
-msgstr ""
+msgstr "Перенести обʼєкти та посилання за архівом"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Перенести архів обʼєктів та посилань"
```

File: po/uk.po [hunk 18]
```diff
@@ -16914,163 +16933,165 @@ msgstr "неприпустиме значення кольору: %.*s"
...
 
 #: command-list.h:65
 msgid "Provide content or type and size information for repository objects"
-msgstr ""
+msgstr "Надавати вміст або інформацію про тип і розмір обʼєктів сховища"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Показати вміст або інформацію про тип і розмір обʼєктів сховища"
```

File: po/uk.po [hunk 19]
```diff
@@ -16914,163 +16933,165 @@ msgstr "неприпустиме значення кольору: %.*s"
...
 
 #: command-list.h:66
 msgid "Display gitattributes information"
-msgstr ""
+msgstr "Відображати інформацію про gitattributes"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Відобразити інформацію про gitattributes"
```

File: po/uk.po [hunk 20]
```diff
@@ -16914,163 +16933,165 @@ msgstr "неприпустиме значення кольору: %.*s"
...
 
 #: command-list.h:67
 msgid "Debug gitignore / exclude files"
-msgstr ""
+msgstr "Налагодити gitignore / вилучити файли"
 
 #: command-list.h:68
 msgid "Show canonical names and email addresses of contacts"
-msgstr ""
+msgstr "Показувати канонічні імена та адреси електронної пошти контактів"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Показати канонічні імена та адреси електронної пошти контактів"
```

File: po/uk.po [hunk 21]
```diff
@@ -16914,163 +16933,165 @@ msgstr "неприпустиме значення кольору: %.*s"
...
 
 #: command-list.h:71
 msgid "Copy files from the index to the working tree"
-msgstr ""
+msgstr "Скопіювати файли з індексу в робоче дерево"
 
 #: command-list.h:72
 msgid "Find commits yet to be applied to upstream"
 msgstr ""
+"Знайдіть коміти, які ще не були застосовані до першоджерельного сховища"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"Знайти коміти, які ще не були застосовані до першоджерельного сховища"
```

File: po/uk.po [hunk 22]
```diff
@@ -16914,163 +16933,165 @@ msgstr "неприпустиме значення кольору: %.*s"
...
 
 #: command-list.h:76
 msgid "Clone a repository into a new directory"
-msgstr ""
+msgstr "Клонувати сховище в нову директорію"
 
 #: command-list.h:77
 msgid "Display data in columns"
-msgstr ""
+msgstr "Відображати дані в стовпчиках"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Відобразити дані в стовпчиках"
```

File: po/uk.po [hunk 23]
```diff
@@ -16914,163 +16933,165 @@ msgstr "неприпустиме значення кольору: %.*s"
...
 
 #: command-list.h:85
 msgid "Helper to store credentials on disk"
-msgstr ""
+msgstr "Помічник для зберігання облікових даних на диску"
 
 #: command-list.h:86
 msgid "Export a single commit to a CVS checkout"
-msgstr ""
+msgstr "Експортувати окремий коміт до CVS переходу"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Експортувати окремий коміт до CVS стану"
```

File: po/uk.po [hunk 24]
```diff
@@ -16914,163 +16933,165 @@ msgstr "неприпустиме значення кольору: %.*s"
...
 
 #: command-list.h:88
 msgid "A CVS server emulator for Git"
-msgstr ""
+msgstr "Емулятор CVS сервера для Git"
 
 #: command-list.h:89
 msgid "A really simple server for Git repositories"
-msgstr ""
+msgstr "Дуже простий сервер для Git-сховищ"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Дуже простий сервер для Git сховищ"
```

File: po/uk.po [hunk 25]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
 #: command-list.h:95
 msgid "Compares the content and mode of blobs found via two tree objects"
 msgstr ""
+"Порівнює вміст та режим blob обʼєктів, знайдених за допомогою двох "
+"деревовидних обʼєктів"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"Порівнює вміст та режим blob обʼєктів, знайдених у двох "
"обʼєктах дерева"
```

File: po/uk.po [hunk 26]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
+msgstr "Отримати відсутні обʼєкти з іншого сховища"
 
 #: command-list.h:101
 msgid "Rewrite branches"
 msgstr "Перезаписати гілки"
 
 #: command-list.h:102
 msgid "Produce a merge commit message"
-msgstr ""
+msgstr "Створити допис до коміту"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Створити допис до коміту злиття"
```

File: po/uk.po [hunk 27]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
 
 #: command-list.h:106
 msgid "Verifies the connectivity and validity of the objects in the database"
-msgstr ""
+msgstr "Перевіряє звʼязність та валідність обʼєктів у базі даних"
 
 #: command-list.h:107
 msgid "Cleanup unnecessary files and optimize the local repository"
-msgstr ""
+msgstr "Видалення непотрібних файлів та оптимізація локального сховища"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Видалити непотрібні файли та оптимізувати локальне сховище"
```

File: po/uk.po [hunk 28]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
 
 #: command-list.h:108
 msgid "Extract commit ID from an archive created using git-archive"
 msgstr ""
+"Вилучити ідентифікатор коміту з архіву, створеного за допомогою git-archive"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"Витягти ідентифікатор коміту з архіву, створеного за допомогою git-archive"
```

File: po/uk.po [hunk 29]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
 
 #: command-list.h:117
 msgid "Send a collection of patches from stdin to an IMAP folder"
-msgstr ""
+msgstr "Надіслати збірку латок зі stdin до теки IMAP"
 
 #: command-list.h:118
 msgid "Build pack index file for an existing packed archive"
-msgstr ""
+msgstr "Створити індексний файл пакунка для існуючого запакованого архіву"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Побудувати індексний файл пакунка для існуючого запакованого архіву"
```

File: po/uk.po [hunk 30]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
 
 #: command-list.h:119
 msgid "Create an empty Git repository or reinitialize an existing one"
-msgstr ""
+msgstr "Створити порожнє Git сховище або переініціалізувати існуюче"
 
 #: command-list.h:120
 msgid "Instantly browse your working repository in gitweb"
-msgstr ""
+msgstr "Миттєво переглядати своє робоче сховище в gitweb"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Миттєво переглянути своє робоче сховище в gitweb"
```

File: po/uk.po [hunk 31]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
+msgstr "Показати інформацію про файли в індексі та робочому дереві"
 
 #: command-list.h:124
 msgid "List references in a remote repository"
 msgstr "Показати посилання у віддаленому сховищі"
 
 #: command-list.h:125
 msgid "List the contents of a tree object"
-msgstr ""
+msgstr "Вивести вміст деревовидного обʼєкту"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Показати вміст обʼєкта дерева"
```

File: po/uk.po [hunk 32]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
 
 #: command-list.h:126
 msgid "Extracts patch and authorship from a single e-mail message"
-msgstr ""
+msgstr "Вилучає латку та авторство з одного повідомлення електронної пошти"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Витягує латку та авторство з одного повідомлення електронної пошти"
```

File: po/uk.po [hunk 33]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
 
 #: command-list.h:127
 msgid "Simple UNIX mbox splitter program"
-msgstr ""
+msgstr "Проста програма-розгалужувач UNIX mbox"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Проста програма-розщеплювач UNIX mbox"
```

File: po/uk.po [hunk 34]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
 
 #: command-list.h:128
 msgid "Run tasks to optimize Git repository data"
-msgstr ""
+msgstr "Запустити завдання для оптимізації даних Git сховища"
 
 #: command-list.h:129
 msgid "Join two or more development histories together"
-msgstr ""
+msgstr "Обʼєднати дві або більше історій розвитку"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Обʼєднати дві або більше історій розробки"
```

File: po/uk.po [hunk 35]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
 
 #: command-list.h:136
 msgid "Creates a tag object with extra validation"
-msgstr ""
+msgstr "Створює обʼєкт тегу з додатковою перевіркою"
 
 #: command-list.h:137
 msgid "Build a tree-object from ls-tree formatted text"
-msgstr ""
+msgstr "Побудувати обʼєкт дерева з тексту у форматі ls-дерева"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Побудувати обʼєкт дерева з ls-tree форматованого тексту"
```

File: po/uk.po [hunk 36]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
 
 #: command-list.h:138
 msgid "Write and verify multi-pack-indexes"
-msgstr ""
+msgstr "Написати і перевірити multi-pack-indexes"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Записати і перевірити multi-pack-indexes"
```

File: po/uk.po [hunk 37]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
 
 #: command-list.h:139
 msgid "Move or rename a file, a directory, or a symlink"
-msgstr ""
+msgstr "Перемістити або перейменувати файл, директорію або символьне посилання"
 
 #: command-list.h:140
 msgid "Find symbolic names for given revs"
-msgstr ""
+msgstr "Знайти символічні назви для заданих revs"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Знайти символьни назви для заданих ревізій"
```

File: po/uk.po [hunk 38]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
 
 #: command-list.h:141
 msgid "Add or inspect object notes"
 msgstr "Додайте або перевірте нотатки до обʼєктів"
 
 #: command-list.h:142
 msgid "Import from and submit to Perforce repositories"
-msgstr ""
+msgstr "Імпортувати з та надсилати до Perforce сховищ"
```
Comment (arkid15r) [inline po/uk.po:17295]:
```suggestion
msgstr "Імпортувати з та надсилати до Perforce сховищ"
```

File: po/uk.po [hunk 39]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
+msgstr "Перемістити або перейменувати файл, директорію або символьне посилання"
 
 #: command-list.h:140
 msgid "Find symbolic names for given revs"
-msgstr ""
+msgstr "Знайти символічні назви для заданих revs"
 
 #: command-list.h:141
 msgid "Add or inspect object notes"
 msgstr "Додайте або перевірте нотатки до обʼєктів"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Додати або перевірити нотатки до обʼєктів"
```

File: po/uk.po [hunk 40]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
 
 #: command-list.h:149
 msgid "Fetch from and integrate with another repository or a local branch"
-msgstr ""
+msgstr "Отримати та інтегрувати з іншим сховищем або локальною гілкою"
 
 #: command-list.h:150
 msgid "Update remote refs along with associated objects"
-msgstr ""
+msgstr "Оновити віддалені рефи разом з асоційованими обʼєктами"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Оновити віддалені посилання разом з асоційованими обʼєктами"
```

File: po/uk.po [hunk 41]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
 
 #: command-list.h:153
 msgid "Reads tree information into the index"
-msgstr ""
+msgstr "Зчитує інформацію про дерево в індекс"
 
 #: command-list.h:154
 msgid "Reapply commits on top of another base tip"
-msgstr ""
+msgstr "Застосувати коміти поверх іншої базової верхівки"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Застосувати коміти поверх іншої верхівки бази"
```

File: po/uk.po [hunk 42]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
 
 #: command-list.h:158
 msgid "Pack unpacked objects in a repository"
-msgstr ""
+msgstr "Запакувати розпаковані обʼєкти у сховищі"
 
 #: command-list.h:159
 msgid "Create, list, delete refs to replace objects"
-msgstr ""
+msgstr "Створити, показати, видалити посилання для заміни об’єктів"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Створити, показати, видалити посилання для об’єктів заміни"
```

File: po/uk.po [hunk 43]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
 
 #: command-list.h:155
 msgid "Receive what is pushed into the repository"
-msgstr ""
+msgstr "Отримати те, що надсилається до сховища"
 
 #: command-list.h:156
 msgid "Manage reflog information"
-msgstr ""
+msgstr "Керування reflog інформацією"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Керування інформацією логу посилань"
```

File: po/uk.po [hunk 44]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
 
 #: command-list.h:159
 msgid "Create, list, delete refs to replace objects"
-msgstr ""
+msgstr "Створити, показати, видалити посилання для заміни об’єктів"
 
 #: command-list.h:160
 msgid "Generates a summary of pending changes"
-msgstr ""
+msgstr "Створює підсумок невирішених змін"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Створює підсумок змін для розгляду"
```

File: po/uk.po [hunk 45]
```diff
@@ -17079,270 +17100,275 @@ msgstr "Порівняти дерево з робочим деревом або
...
 
 #: command-list.h:161
 msgid "Reuse recorded resolution of conflicted merges"
-msgstr ""
+msgstr "Повторне використання записаного розв’язання конфліктних злиттів"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "Використати повторно записане розв’язання конфліктних злиттів"
```

File: po/uk.po (hunk 7 above)
Comment (kasya) [reply po/uk.po]: 👍🏼

File: po/uk.po (hunk 38 above)
Comment (kasya) [reply po/uk.po:17295]: Is there a difference?🤔

Comment (arkid15r) [reply po/uk.po:17295]: 🤷

**Pull request merged**